### ISSUE TITLE
[Meta] English or German

### ISSUE THREAD

**Issue by chrisaut** [portfolio-performance/portfolio#1671]
I notice a lot of issues/discussions here get defaulted to German, and while I recognize that the vast majority of PP users are german speaking I truly believe this puts a barrier to having developers join in who do not speak the language.
Of course, if someone is not comfortable enough to communicate in English, they can always fall back to German, but do you think that we should make an effort to be more international and welcoming to non-German speakers @buchen ?

**Comment by buchen** [comment]
Fair point. I try to keep the whole source code in English (apart from the German word here and there in the PDF imports of the banks). But for the source code, the issues and pull request, I am not very consequent so far. But I totally see your point. With the contributions of translations for Dutch, Spanish, Portuguese, and - upcoming - French, it makes sense to have the contributions and discussions in English. I'll vow to be better. :-)

**Comment by fbuchinger** [comment]
@buchen it would be great if you communicate your language preference to all bug/feature request reporters via [GitHub Issue Templates](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository). By doing so, we can avoid German-only tickets and increase the number of potential contributors.
